conformance/lang/expressions/multiplicative-expr/multiplication.balt

```diff
@@ -0,0 +1,3340 @@
+Test-Case: output
+Description: Test integer multiplication non-overflow scenarios, for positive non-zero integers.
+Labels: multiplicative-expr, int, DecimalNumber
+
+public function main() {
+    int a = 1;
+    int b = 2;
+    int c = 4611686018427387903;
+    int d = 6917529027641081853;
+    int e = 9223372036854775807;
```

Comment:
Shall we add 1,2 tests withHexIntLiterals as well?
```ballerina
public function main() {
    int a = 0xfac * 0xabc;
}
```

Reply:
Added in 8e518b3.